apps/site/pages/en/learn/modules/package-configuration.md

```diff
@@ -0,0 +1,408 @@
+---
+title: Package Configuration
+layout: learn
+authors: JakobJingleheimer
+---
+
+# Package configuration
+
+Configuration is always a chore, but an unfortunately necessary evil. And configuring a package for CommonJS (CJS) and ES Modules (ESM) can be a waking nightmare—not least because it has changed a dozen times in half as many years.
```

Comment:
```suggestion
Configuring a node.js project/package is always a chore, but an unfortunate necessary evil. And configuring a package for CommonJS (CJS) and ES Modules (ESM) can be a waking nightmare - not least because it has changed a dozen times in half as many years.
```

Reply:
i dont think this is better in this case

Reply:
I think unfortunately -> unfortunate is better (seeing as unfortunate is an adjective, rather than an adverb).